sorc/link_fv3gfs.sh

```diff
@@ -57,6 +57,8 @@ elif [ $machine = "orion" ]; then
     else
        FIX_DIR="/work/noaa/global/glopara/fix_nco_gfsv16"
     fi
+elif [ $target == "stampede" ]; then
```

Comment:
These fix files should be pointing to a. fix_UFSp6

Reply:
I will update it.